src/main/resources/db/migration/V20220216_66__HMAN-66-Add_Column_PartiesNotifiedDateTime.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ONLY public.hearing_response
```

Comment:
if you are merging both HMAN-66 and 67 together - do we need this file when this column is added in [V20220214_51__HMAN-51-PartiesNotified.sql](https://github.com/hmcts/hmc-cft-hearing-service/pull/78/files#diff-b9b01ebe8f881acf5e029fac5f442cd6a6af181c5eafff01c084d10835fe96e9)

Reply:
Good point